CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed: *  Comments lost from multi-line expressions [#2525](https://github.com/fsprojects/fantomas/issues/2525)
```

Comment:
Please revisit this. Look very closely at how the rest of the file was set up.
Put the entry on the following line, remove unnecessary spaces and add a `.` after the issue title.